apps/consulting/utils/getArticlesPaths/getArticlesPaths.ts

```diff
@@ -2,14 +2,14 @@ import { TLinkEntry } from '@quansight/shared/types';
 
 import { LinkEntry } from '../../api/types/basic';
 import { formatSlugParam } from './formatSlugParam';
-import { isSlugRestricted } from './isSlugRestricted';
+import { isSlugForPost } from './isSlugForPost';
```

Comment:
It looks like this is a cosmetic+semantic name change, to differentiate this from the `isSlugRestricted` elsewhere in the PR?

Reply:
It's not to differentiate. Given what the function does, the name simply made no sense to me, so I decided to change it.

Reply:
Differentiation is just a side benefit 😃